.travis.yml

```diff
@@ -5,7 +5,7 @@ python:
   - "3.2"
```

Comment:
IPython requires Python version 2.7 or 3.3 or above. So you have to remove 3.2 option.